lib/v3/index.mjs

```diff
@@ -32,3 +33,8 @@ export {
   isVue3,
   install,
 }
+
+//Fix hasInjectionContext is not defined https://github.com/vueuse/vue-demi/issues/233
```

Comment:
v3 already has the function so this should be removed

Reply:
![image](https://github.com/vueuse/vue-demi/assets/13673962/d0a8fd40-3da4-4614-9aaa-3e43d3dbc918)

But I did not found that function in v3.3.4, and so much programer to feedback this [question](https://github.com/vueuse/vue-demi/issues/233)

Reply:
https://github.com/vueuse/vue-demi/issues/233#issuecomment-1554324039